src/DataViews/include/DataViews/PresetsDataView.h

```diff
@@ -18,7 +18,12 @@
 #include "PresetFile/PresetFile.h"
 #include "preset.pb.h"
 
+namespace orbit_data_views_internal {
+std::string FormatShortDatetime(absl::Time time);
+}  // namespace orbit_data_views_internal
```

Comment:
if you want to make it DataView component internal you can move it to a private header. Private headers are header files located in src/DataView. This way they are not accessible for other components but can still be tested.

Reply:
Thanks! I moved this to `DataViewUtils.h` file and added a test for it